lib/projects/buildAndDeploy.ts

```diff
@@ -169,25 +171,37 @@ function makePollTaskStatusFunc<T extends ProjectTask>({
 
     const subtasks = getSubtasks(initialTaskStatus);
 
-    const tasksById = subtasks.reduce(
-      (acc: { [key: string]: ProjectSubtask }, subtask) => {
+    const hiddenComponentBuildIds: string[] = [];
+
+    const tasksById = subtasks
+      .filter(subtask => {
+        // TODO: Remove this filtering logic when visible=false for SERVERLESS_PACKAGE
+        const shouldBeVisible =
+          getSubtaskType(subtask) !== mapToInternalType(AppFunctionsPackageKey);
+        if (!shouldBeVisible) {
+          hiddenComponentBuildIds.push(subtask.id);
+        }
+        return shouldBeVisible;
+      })
```

Comment:
This is a hopefully temporary solution to prevent the `serverless-package` component from showing up in the loading spinners.